SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -694,7 +711,9 @@ Worker_ComponentUpdate USpatialSender::CreateMulticastUpdate(UObject* TargetObje
 		}
 	}
 
-	AddBytesToSchema(EventData, 1, PayloadWriter);
+	Schema_AddUint32(EventData, 1, TargetObjectRef.Offset);
```

Comment:
This code is duplicated. Can we make a function that takes in a Schema_Object and writes an RPC payload?